node/tests/ui_gateway_test.rs

```diff
@@ -166,7 +166,7 @@ fn daemon_does_not_allow_node_to_keep_his_client_alive_integration() {
     let assertion_lookup_pattern_2 =
         |_port_spec_ui: &str| "Received shutdown order from client 1".to_string();
     let second_port = connected_and_disconnected_assertion(2, assertion_lookup_pattern_2);
-    //TODO Create card to Change infrastructure and add node_client.send here and it should return result - we send the suhtdown request in loop until it not returns
+    //TODO Card #806 "create looping utility with node_client.send": Create a test utility with a node_client.send(shutdown_request) and keep sending the shutdown request in a loop until the Node doesn't response anymore.
```

Comment:
You can possibly leave just the name of the name of the card here, including the numeric marking `806`. The rest can be read when you find the card. 

Or better, something saying: "Test utility to easily verify the Node's termination"